system/lib/libc/emscripten_memcpy.c

```diff
@@ -9,20 +9,20 @@
 #include "emscripten_internal.h"
 
 // Use the simple/naive version of memcpy when building with asan
-#if defined(EMSCRIPTEN_OPTIMIZE_FOR_OZ) || __has_feature(address_sanitizer)
+#if __has_feature(address_sanitizer)
 
 static void *__memcpy(void *dest, const void *src, size_t n) {
   unsigned char *d = (unsigned char *)dest;
   const unsigned char *s = (const unsigned char *)src;
-#pragma clang loop unroll(disable)
   while(n--) *d++ = *s++;
   return dest;
 }
 
-#elif defined(__wasm_bulk_memory__)
+#elif defined(__wasm_bulk_memory__) || defined(EMSCRIPTEN_OPTIMIZE_FOR_OZ)
```

Comment:
Won't `__wasm_bulk_memory__` always be defined now making the first clause redundant?

Reply:
Yeah, as you mention in the comment below, that would make this file uncompileable without bulk memory but maybe that's fine?

Reply:
Thats fine I think.  I don't imagine we want to support anyone trying do a custom compile for libc without bulk memory after this change.